packages/salesforcedx-vscode-apex/snippets/apex.json

```
@@ -410,5 +409,54 @@
             "}"
         ],
         "description": "Create a mock for an http callout for apex testing"
+    },
+    "Is Blank": {
+        "prefix": "isb",
+        "body": [
+            "String.isBlank(${0})"
+        ],
+        "description": "String isBlank"
+    },
+    "Is Not Blank": {
+        "prefix": "isnb",
+        "body": [
+            "String.isNotBlank(${0})"
+        ],
+        "description": "String Not isBlank"
+    },
+    "local variable": {
+        "prefix": "localVariable",
+        "body": [
+            "${1:Object} ${2:name} = ${3:};"
+        ],
+        "description": "New local variable"
+    },
+    "static variable": {
+        "prefix": "staticVariable",
+        "body": [
+            "${1|private,public,global|} static ${2:Object} ${3:name} = ${4:};"
+        ],
+        "description": "New static variable"
+    },
+    "variable": {
+        "prefix": "variable",
+        "body": [
+            "${1|private,public,global|} ${2:Object} ${3:name} = ${4:};"
+        ],
+        "description": "New variable"
+    },
+    "static final variable": {
+        "prefix": "staticFinalVariable",
+        "body": [
+            "${1|private,public,global|} static final ${2:Object} ${3:name} = ${4:};"
+        ],
+        "description": "New static final variable"
+    },
+    "System Debug 2": {
+      "prefix": "sd",
+      "body": [
+          "System.debug('{$0:data} >>' + {$0:data});"
```

Is my proposed change what you meant instead?  I moved the left curly brace to the right of the dollar sign to avoid a syntax error caused by the presence of the curly braces and colon.  Also, "varName" is clearer than "data" for a placeholder of a variable.

Previous snippet: `System.debug('{:data} >>' + {:data});`
New snippet: `System.debug('varName >>' + varName);`

```suggestion
          "System.debug('${0:varName} >>' + ${0:varName});"
```